src/main/java/com/massivecraft/factions/cmd/CmdShow.java

```diff
@@ -69,18 +76,18 @@ public void perform() {
             double refund = value * Conf.econClaimRefundMultiplier;
             if (value > 0.0D) {
                 String stringValue = Econ.moneyString(value);
-                String stringRefund = refund > 0.0D ? " (" + Econ.moneyString(refund) + " depreciated)" : "";
-                msg("<a>Total land value: <i>" + stringValue + stringRefund);
+                String stringRefund = (refund > 0.0) ? (TL.COMMAND_SHOW_DEPRECIATED.format(Econ.moneyString(refund))) : "";
+                msg(TL.COMMAND_SHOW_LANDVALUE, stringValue, stringRefund);
             }
 
             if (Conf.bankEnabled) {
-                msg("<a>Bank contains: <i>" + Econ.moneyString(Econ.getBalance(faction.getAccountId())));
+                msg(TL.COMMAND_SHOW_BANKCONTAINS, Econ.moneyString(Econ.getBalance(faction.getAccountId())));
             }
 
         }
 
-        String allyList = p.txt.parse("<a>Allies: ");
-        String enemyList = p.txt.parse("<a>Enemies: ");
+        String allyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ALLIES);
+        String enemyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ENEMIES);
```

Comment:
We wouldn't need to have the text parsing as well as the <a> if we are using TL.